common/src/main/java/com/wynntils/services/mapdata/MapAttributesResolver.java

```diff
@@ -66,33 +69,44 @@ private <T> T getAttribute(Function<MapAttributes, Optional<T>> attributeGetter)
         return attributeGetter.apply(DefaultMapAttributes.INSTANCE).get();
     }
 
+    private ResolvedMarkerOptions getResolvedMarkerOptions(
+            Function<MapAttributes, Optional<MarkerOptions>> attributeGetter) {
+        return new ResolvedMarkerOptions(
+                getInheritedValue(MarkerOptions::getInnerRadius, attributeGetter),
+                getInheritedValue(MarkerOptions::getOuterRadius, attributeGetter),
+                getInheritedValue(MarkerOptions::getFade, attributeGetter),
+                getInheritedValue(MarkerOptions::getBeaconColor, attributeGetter),
+                getInheritedValue(MarkerOptions::labelVisible, attributeGetter),
+                getInheritedValue(MarkerOptions::distanceVisible, attributeGetter),
+                getInheritedValue(MarkerOptions::iconVisible, attributeGetter));
```

Comment:
I'm still not satisfied (I know I suggested this myself, yeah yeah :-D).

What about `hasIcon` and `hasLabel`? I think that would be the best.

Reply:
I've renamed the two, and left `distanceVisible` explicitly as is, as `hasDistance` doesn't make much sense.

Reply:
And I now see the next comment. Don't mind me.

Reply:
The new `has` names are much better. It needs to be updated everywhere. I assume the getter will need to be `getHasLabel` etc, even though it sounds a bit ungrammatical. The alternative is `isHasLabel` (javabean-style getter) or just `hasLabel`, but then that will look like a record-style getter with no prefix, which will be inconsistent.